KomaMRIBase/Project.toml

```diff
@@ -5,11 +5,13 @@ version = "0.8.1"
 
 [deps]
 Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MAT = "23992714-dd62-5051-b70f-ba57cb901cac"
 MRIBase = "f7771a9a-6e57-4e71-863b-6e4b6a2f17df"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
```

Comment:
Try not to add new packages

Reply:
Resolved in [7f1cc52](https://github.com/JuliaHealth/KomaMRI.jl/pull/184/commits/7f1cc52925d3be8803c4ff0c95c1e06f391b7c83)